frontend/src/metabase-lib/types.ts

```diff
@@ -141,6 +141,14 @@ export type ColumnDisplayInfo = {
   selected?: boolean; // used in aggregation and field clauses
 };
 
+export type SegmentDisplayInfo = {
+  name: string;
+  displayName: string;
+  longDisplayName: string;
+  description: string;
+  effectiveType: string;
+};
```

Comment:
Do we really need the `effectiveType` here? I'd rather prefer if the FE didn't know about it at all

Reply:
I just typed it because MLv2 gives it to us. 🤷

Reply:
I think we can skip it 👍